.gitignore

```diff
@@ -95,3 +95,9 @@ dump.rdb
 requirements/node*
 requirements/yarn*
 /node
+
+# postgres database
+postgres/
+# safeguarding secrets
+sfdx_hub.key
+/app/sfdx_hub.key
```

Comment:
You shouldn't need this last line; the previous one includes the file regardless of what path it's in.